First draft of the Finance Section (#31)
astropy/astropy-v5.0-paper: src/ms.tex

\secauthor{Aarya Patil}

% Summarize funding sources (Moore, NASA) and amounts and what this has been
% used for.

With the goal of establishing a sustainable financial model for the Astropy

> project, the Coordination Committee under the new governance charter (see \S

Reviewer: ```suggestion
Project, the Coordination Committee under the new governance charter (see \S
```

\ref{sec:project-governance}) established a standing Finance committee.

> This committee overlooks the planning and disbursement of finances on behalf

Reviewer: ```suggestion
This committee oversee planning and allocation of finances on behalf
```
maybe it's just me, but "disbursement" seems a bit less community-centric, and this sounds better to me

> of the project. The Astropy project accepts funds from institutions as well as

Reviewer: ```suggestion
of the project. The Astropy Project accepts funds from institutions as well as
```

> individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that

Reviewer 2: NUMFOCUS capitalization.

> supports and promotes world-class, innovative, open source scientific computing.},
> the fiscal sponsor of the project. Previously, no direct financial support

Reviewer: ```suggestion
supports and promotes world-class, innovative, open source scientific computing.}.
Previously, no direct financial support
```

I know this is what NF says themselves, but I think "fiscal sponsor" is often prone to misinterpretation as though they are giving us money as opposed to being more like the "bank".  (this goes with another comment below about NF).

> was available for the project development, and NUMFOCUS covered the incurred

Reviewer 2: NumFOCUS didn't cover all (or any?) incurred costs.

Author: I took this from the Astropy v2.0 paper: https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f/pdf -- 2.6. Support of Astropy "NumFOCUS has, to date, covered the direct costs incurred by the Astropy Project." Is that not correct? Should I use the word "most" or "some"?

operational costs. However, transitioning to long-term financial stability meant

> having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},

Reviewer 2: Remove citation.

> and hence the Astropy project applied for and successfully acquired several grants.

Reviewer: ```suggestion
and hence the Astropy Project applied for and successfully acquired several grants.
```

> and hence the Astropy project applied for and successfully acquired several grants.

Reviewer 2: Is “several” == “four”? I feel like if we know there are four grants we can enumerate below, we should say that.

> The two major ones are briefly summarized below.

Reviewer: (to combine with the removal of "fiscal sponsor" above)
```suggestion
These grants are administered by NumFOCUS on behalf of the project, providing
a ``neutral'' space not tied to any particular astronomy institution.
The two major ones are briefly summarized below.
```

\begin{itemize}
    \item Moore: The Gordon and Betty Moore Foundation awarded the Astropy

>     project a \$900,000 grant in 2019 for the proposal ``Sustaining and Growing

Reviewer 2: \$900,000 (US).

Reviewer 2: Also, make the figures approximate (“~$900k”)

    the Astropy Project". Over a three-year term, this grant supported sustenance
    of the project by providing funds for (1) the development and maintenance of
    the \astropypkg and its infrastructure to existing project members and
    targeted hires, (2) the transition of long-term users to contributors and
    maintainers through mentorship efforts, (3) the formalization of a governance
    structure (see \S \ref{sec:project-governance}), and (4) the improvement
    of equity, diversity, and inclusion efforts in the project (see \S
    \ref{sec:project-ide}). Some of the money supported travel for meetings,
    conferences, and workshops. This generous grant helped reduce reliance on
    volunteer-driven maintenance of the openly-developed package and paved
    the way to acquire funding from federal agencies: a big milestone for the
    success of the project.

>     \item NASA: A successful proposal to the NASA ROSES-2020 call, section E.7

Reviewer 2: Footnote with reference to the call?

    (Support for Open Source Tools, Frameworks, and Libraries), provided the

>     Astropy project \$634,590 to support its work. Funded work for a term of

Reviewer 2: “~$600k”

    three years includes (1) project-wide infrastructure maintenance and
    improvements, (2) targeted work on areas of the \astropy core and
    coordinated packages that need support, (3) enhancements to the Learn
    Astropy ecosystem and educational materials (see \S \ref{sec:learn}),
    and (4) support for the \astropy affiliated packages. These goals
    support the sustenance of the project by encouraging further
    development of the package ecosystem and community engagement.
\end{itemize}

In addition to the two grants described above, the project benefited from

> a Gemini Observatory contract under its Science User Support Department,

Reviewer 2: Finance Ctte are going to run these two sentences past Gemini and Dunlap just to confirm they are happy with these mentions.

Reviewer 3: @jdswinbank Did you hear back?

Reviewer 2: As I recall this was an action on @eteq, so perhaps he could commet.

Reviewer 3: I heard from @eteq in Slack that Gemini is happy as long as we add their standard acknowledgement -- I'll merge this and add the ack in a follow-up commit

“Supported by the international Gemini Observatory, a program of NSF’s NOIRLab, which is managed by the Association of Universities for Research in Astronomy (AURA) under a cooperative agreement with the National Science Foundation, on behalf of the Gemini partnership of Argentina, Brazil, Canada, Chile, the Republic of Korea, and the United States of America.”

which funded development work to support both the Astropy and DRAGONS

> projects. The project also received a total of CAD 31,023 from the Dunlap

Reviewer 2: $31,023 (Canadian).

Reviewer 2: Better: remove the figure.
